packages/temporal/mpe/shared/mpe.go

```diff
@@ -82,6 +82,16 @@ func (s *TrackMetadataSet) Clear() {
 	s.tracks = []shared.TrackMetadata{}
 }
 
+func (s *TrackMetadataSet) GetTotalTracksDuration() int64 {
+	var totalDuration int64 = 0
+
+	for _, track := range s.tracks {
+		totalDuration += track.Duration.Milliseconds()
+	}
+
+	return totalDuration
+}
```

Comment:
In JavaScript, would you have written this with a `Array.reduce`? 👀

Reply:
I would'nt reduce myself to that kind of pratices :laughing: